base/v0_2/translate_test.go

```diff
@@ -196,7 +196,8 @@ func TestTranslateFile(t *testing.T) {
 							},
 						},
 						{
-							Source: util.StrToPtr("data:,file%20contents%0A"),
+							Source:      util.StrToPtr("data:,file%20contents%0A"),
+							Compression: util.StrToPtr(""),
```

Comment:
Do we actually need to change all the tests to explicitly specify a `""` compression?  Shouldn't it be equivalent to `nil`?

Reply:
Right, the whole point of this PR is to emit `""` rather than `nil` because they're semantically distinct for config merging.

Reply:
> Right, the whole point of this PR is to emit "" rather than nil because they're semantically distinct for config merging.

Yeah sorry, it is obvious when stated that way.  I don't know why I find this so confusing.